### ISSUE TITLE
Synchronous operations are disallowed in ASP.NET Core 3.0.0

### ISSUE THREAD

**Issue by vertonghenb** [arichika/RotativaCore#5]
Is it possible to create the .Pdf files async instead of sync? In ASP.NET Core 3.0.0 it's no longer allowed by default and the action will throw:

Issues start when calling new ViewAsPdf()

```cs 
InvalidOperationException: Synchronous operations are disallowed. Call WriteAsync or set AllowSynchronousIO to true instead.
```

Announcement can be found here: https://github.com/aspnet/AspNetCore/issues/7644

**Comment by arichika** [comment]
@vertonghenb  Thank you very much. I want to fix it.

**Comment by vertonghenb** [comment]
@arichika PR is ready to fix this issue. However there are still some blocking .Wait() calls that could be fixed in the later run.

**Comment by arichika** [comment]
Closed #6

**Comment by arichika** [comment]
Rreleased and Updated the README. Thank you. @vertonghenb 
https://www.nuget.org/packages/RotativaCore/3.0.0
